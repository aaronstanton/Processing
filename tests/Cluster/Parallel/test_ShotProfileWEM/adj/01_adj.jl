using Seismic

# initialize only 1 process per node
Seismic.bind_torque_procs(1)

@everywhere using Seismic

# for each node, migrate 1 common shot gather using 12 threads per node. 
param = ["adj"=>"y",
    "vel"=>"../prep/vp", "wav"=>"../prep/wav",
    "angx"=>"../prep/angx","angy"=>"../prep/angy",
    "pade_flag"=>"y",
    "sz"=>10, "gz"=>550,
    "fmin"=>1., "fmax"=>50.,
    "padt"=>2, "padx"=>2,
    "nangx"=>71, "oangx"=>-70, "dangx"=>2,
    "nangy"=>1, "oangy"=>0, "dangy"=>2,
    "nhx"=>827, "ohx"=>-3304, "dhx"=>8,
    "nhy"=>1, "ohy"=>0, "dhy"=>8,
    "omp"=>12,"verbose"=>"y"];
param["sx"] = [100.0:100.0:6900.0];
param["sy"] = [100.0:100.0:6900.0]*0;
ShotProfileWEM("../prep/uz","mpp",param);

