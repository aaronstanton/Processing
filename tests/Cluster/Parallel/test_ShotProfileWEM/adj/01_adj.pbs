#!/bin/bash 
#PBS -S /bin/bash
#PBS -l nodes=3:ppn=12 
#PBS -l mem=66gb 
#PBS -l walltime=2:00:00 
#PBS -m bea 
#PBS -M aaron.stanton@ualberta.ca
#PBS -N 01_adj
#PBS -o 01_adj.out
#PBS -e 01_adj.err

date

# Change to directory from which job was submitted
cd $PBS_O_WORKDIR 

echo "here is your PBS_NODEFILE"
cat $PBS_NODEFILE

echo "calling julia now"
julia 01_adj.jl

