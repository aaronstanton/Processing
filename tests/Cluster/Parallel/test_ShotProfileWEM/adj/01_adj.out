Mon Aug 17 12:44:28 MDT 2015
here is your PBS_NODEFILE
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0789
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0798
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
cn0805
calling julia now
name of compute nodes and number of workers:
["cn0805"=>12,"cn0789"=>12,"cn0798"=>12]
added 1 processes on master itself
added 1 processes on machine cn0805
added 1 processes on machine cn0798
done
